scripts/apiStage.sh

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/env bash
```

Comment:
'set -euo pipefail' might help catch errors early and executing subsequent commands when something fails. 

https://explainshell.com/explain?cmd=set%20-euxo%20pipefail